pkg/quarterdeck/db/migrations/0005_user_org_logins.sql

```diff
@@ -1,8 +1,11 @@
--- Add a column to the organization_users to record last login times.
+-- Add columns to the organization_users table.
 
 BEGIN;
 
 -- Last login time for the user in the organization.
 ALTER TABLE organization_users ADD COLUMN last_login TEXT DEFAULT NULL;
 
+-- Delete confirmation token for the organization user.
+ALTER TABLE organization_users ADD COLUMN delete_confirmation_token TEXT DEFAULT NULL;
```

Comment:
Do we need to add this to a new migration since we've already deployed to staging?

Reply:
We can delete the staging database; has this migration been applied to prod?

Reply:
I don't think it has because I just added the other migration a few days ago.